resources/views/common/footer.blade.php

```diff
@@ -0,0 +1,10 @@
+@if(setting('app-privacy-policy') | setting('app-terms-of-service'))
```

Comment:
```suggestion
@if(setting('app-privacy-policy') or setting('app-terms-of-service'))
```
I am not too familiar with blade templates, but without this change, I only get a lot of errors (I only entered a privacy policy link, but not a terms of service)

Reply:
What version of PHP are you running, out of curiosity?

Reply:
PHP 7.3

Reply:
Thinking about it, I guess || would also be possible, but doesn't | do a bitwise or?